0001-template.md

```diff
@@ -86,6 +87,6 @@ Discuss prior art, both the good and the bad, in relation to this proposal.
 ## Unresolved questions
```

Comment:
what do we mean by "art" in:

> ## Prior art
> [prior-art]: #prior-art
> Discuss prior art, both the good and the bad, in relation to this proposal.

Reply:
The term is taken from here: https://en.wikipedia.org/wiki/Prior_art (I think -- we took it from (a very old version of): https://github.com/rust-lang/rfcs/blob/master/0000-template.md )